From e33b0df13c193e0d055ffc2dbf8fcd870e0d37b7 Mon Sep 17 00:00:00 2001
From: Sergii Piatakov <sergii.piatakov@globallogic.com>
Date: Tue, 30 Aug 2022 10:16:08 +0300
Subject: [PATCH 1/3] fix memory overlap for CR7 bootloader

The CR7 bootloader has the same value of start address.

Signed-off-by: Sergii Piatakov <sergii.piatakov@globallogic.com>
---
 board/renesas/rcar-common/common.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/renesas/rcar-common/common.c b/board/renesas/rcar-common/common.c
index 97141587ef..57815b7fa2 100644
--- a/board/renesas/rcar-common/common.c
+++ b/board/renesas/rcar-common/common.c
@@ -249,7 +249,7 @@ static struct img_param img_params[IMG_MAX_NUM] = {
 	[IMG_IPL2] = {
 		.img_id = IMG_IPL2,
 		.img_name = "bl2",
-		.start_addr = 0x40000,
+		.start_addr = 0x140000,
 		.total_size = 1 * HF_SECTOR_SIZE,
 		.sectors_num = 1,
 	},
-- 
2.25.1

